### PR TITLE
PY3: remove print statements

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: tests/test_archiver.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def test_scan_archives_all(self):
         self.archiver.scan_archives()
         archives_for_channel = self.archiver.archives_for_channel
         data = self.archiver.get('XF:23IDA-VA:0{DP:1-IP:1}P-I', '2013-08-11', '2013-08-12')
-        print data
+        print(data)
```
Comment (tacaswell) [inline tests/test_archiver.py]: This also needs the `__future__` imports or you break py2k

Comment (ericdill) [inline tests/test_archiver.py]: @tacaswell print is a function in py27. This breaks py26 without the print_function import though.

**Comment by ericdill** [comment]
@tacaswell Have another look?  Added a few more things and turned on travis.  

Note that print is a function in py27 without the future import. In py26, print is a statement only.

**Comment by ericdill** [comment]
Yeah go ahead and merge if it looks good!

**Comment by ericdill** [comment]
Thanks @dchabot!

**Pull request merged**